src/web-ui/src/public/CategoryDetail.vue

```diff
@@ -108,11 +110,13 @@ export default {
         if (response.headers) {
           if (response.headers['x-personalize-recipe']) {
             this.personalized = true
-            this.explain_recommended = 'Personalize recipe: ' + response.headers['x-personalize-recipe']
+            this.recipe = response.headers['x-personalize-recipe']
+            this.explain_recommended = 'Personalize recipe: ' + this.recipe
           }
           if (response.headers['x-experiment-name']) {
             this.active_experiment = true
-            this.explain_recommended = 'Active experiment: ' + response.headers['x-experiment-name']
+            this.experiment = response.headers['x-experiment-name']
```

Comment:
Is `this.experiment` needed? Doesn't look like it's used anywhere besides the next line.

Reply:
The idea was to have this available in the Amplitude A/B testing workshop which they wanted to add after the experimentation one.

Reply:
> The idea was to have this available in the Amplitude A/B testing workshop which they wanted to add after the experimentation one.

Okay, so another parameter passed to AnalyticsHandler.js calls? If so, why not weave it all the way through now so it's passed to Amplitude (and other event sinks)?

Reply:
It looks like at some point the productViewed AnalyticsHandler function changed to take an experimentCorrelationId vs an experiment parameter.  Just want to make sure this will not be a breaking change as the only call I can see to this function is in the ProductDetail component.